package/boot/uboot-mvebu/Makefile

```diff
@@ -45,6 +53,7 @@ endef
 define Build/InstallDev
 	$(INSTALL_DIR) $(STAGING_DIR_IMAGE)
 	$(CP) $(PKG_BUILD_DIR)/$(UBOOT_IMAGE) $(STAGING_DIR_IMAGE)/$(BUILD_VARIANT)-u-boot-spl.kwb
+	$(CP) $(PKG_BUILD_DIR)/$(UBOOT_IMAGE) $(STAGING_DIR_IMAGE)
```

Comment:
why do you have to copy the file under its original name? This will collide if you build multiple u-boot binaries.

Reply:
Did change name since there is one device so far. Updated the line to copy u-boot to $(BUILD_VARIANT)-u-boot.bin instead.